crates/astria-sequencer/test-genesis-app-state.json

```diff
@@ -27,6 +27,7 @@
     "sequence_byte_cost_multiplier": 1,
     "init_bridge_account_base_fee": 48,
     "bridge_lock_byte_cost_multiplier": 1,
+    "bridge_sudo_change_fee": 24,
```

Comment:
does this make this a hard fork requiring network regen change?

In general actually, we probably need an answer for this. We should be able to add actions with fees without regenesis of network

Reply:
yes, this would be a hard fork. without hard fork logic this is a regenesis